mycroft.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+
+if [ ! -z $1 ]
```

Comment:
This might sit better for people who have muscle memory and automatically type in `start`.
```bash
if [[ ! -z "$1" && "$1" != "start" ]]
then
  echo "stop"
else
  echo "start"
fi
```